.pre-commit-config.yaml

```diff
@@ -24,9 +24,14 @@ repos:
 
 # Attempt to enforce standardized PEP8 style formatting:
 - repo: https://github.com/pre-commit/mirrors-autopep8
-  rev: v1.5.5
+  rev: v1.5.6
   hooks:
   - id: autopep8
+    exclude: ^docs/conf.py$
+    args:
+      - --global-config=tox.ini
+      - --in-place
+      - --max-line-length=88
```

Comment:
I noticed that there was some automated line-wrapping happening, cutting our 88 character lines down to 79 characters, which would generate a bunch of unnecessary diffs. This might be because I consolidated all the linter / formatter configuration files into `tox.ini` Anyway, I couldn't get `pre-commit` to look at `tox.ini` for its configuration (even though that's supposed to work!) so I just forced autopep8 to respect the max line length here.